configure.ac

```diff
@@ -58,13 +58,15 @@ fi
 
 # Checks for libraries.
 AC_SEARCH_LIBS(opendir, [mingwex])
+AC_SEARCH_LIBS(libiconv, [iconv])
```

Comment:
No. It's the correct usage of the library.

Reply:
But, the 1st argument of `AC_SEARCH_LIBS` should be FUNCTION name.
On my Mac environment, it has libiconv.2.4.0.dylib, it has failed to detect iconv library.

```config.log

~~snip~~

| /* end confdefs.h.  */
|
| /* Override any GCC internal prototype to avoid an error.
|    Use char because int might match the return type of a GCC
|    builtin and then its argument prototype would still apply.  */
| #ifdef __cplusplus
| extern "C"
| #endif
| char libiconv ();
| int
| main ()
| {
| return libiconv ();
|   ;
|   return 0;
| }
configure:5228: gcc -o conftest -g -O2   conftest.c -liconv   >&5
Undefined symbols for architecture x86_64:
  "_libiconv", referenced from:
      _main in conftest-d7967a.o
ld: symbol(s) not found for architecture x86_64

~~snip~~
```

Reply:
That's normal for systems without the iconv library, and yes, the library does contain the `libiconv` symbol, not `iconv`. It's so the symbol won't conflict with the C library's implementation of `iconv`.

Reply:
I get it.

Reply:
I imagine that if you built GNU libiconv from source and installed it in a directory that `gcc` could use, that it would be used and the symbols would not be undefined.